src/heap.py

```diff
@@ -38,12 +42,12 @@ def _organize_list(self, lst):
                     lst[i], lst[i * 2 + 2] = lst[i * 2 + 2], lst[i]
             except IndexError:
                 continue
-        print(self.heap)
         for i in range(len(lst)):
+            print(lst)
             try:
                 print('index ', len(lst) - i)
                 if (len(lst) - i) % 2 is 0:
-                    print('if ', lst[(len(lst) - i)], ' compared to ', lst[((len(lst) - i) // 2) - 1])
+                    print('if ', lst[(len(lst) - i)], ' compared to index ', int((len(lst) - i) / 2) - 1, lst[int((len(lst) - i) / 2) - 1])
```

Comment:
Woo, this is not pretty to look at, but it works.  Remove these print statements.  Consider storing these values with a good name, then swapping like chile, parent = parent, child